akka-projection-core/src/main/scala/akka/projection/HandlerRecovery.scala

```diff
@@ -4,17 +4,19 @@
 
 package akka.projection
 
-import akka.util.JavaDurationConverters._
+import scala.annotation.nowarn
 import scala.concurrent.duration.FiniteDuration
 
 import akka.annotation.InternalApi
+import akka.util.JavaDurationConverters._
 
 trait HandlerRecovery[Envelope] {
 
   /**
    * Override this to define the error handler strategy when processing an `Envelope` fails.
    * By default it will immediately give up and fail the stream.
    */
+  @nowarn
```

Comment:
Maybe we have to decide if we are going to support Scala 2.12 or not. Would be nice to only have 2.13 but maybe too early to close the door for 2.12?
See #108